lms/djangoapps/badges/backends/badgr.py

```diff
@@ -30,8 +35,16 @@ class BadgrBackend(BadgeBackend):
 
     def __init__(self):
         super(BadgrBackend, self).__init__()
-        if not settings.BADGR_API_TOKEN:
-            raise ImproperlyConfigured("BADGR_API_TOKEN not set.")
+        if None in (settings.BADGR_USERNAME,
```

Comment:
So the settings all default to `None` in `/lms/envs/common.py`. If I don't add a default `None` value in `common.py`, leave it unset and try to fetch it, I get: `*** AttributeError: 'Settings' object has no attribute 'BADGR_USERNAME'`. So, I thought I was doing things like before, just checking for multiple variables instead of `BADGR_API_TOKEN`. I'm confused now.

Reply:
All right then! I'm probably dead wrong. Just ignore me. :)